.circleci/config.yml

```diff
@@ -35,7 +35,7 @@ jobs:
           name: Pytest Test Cases
           command: | # Run test suite, uses NUCLEUS_TEST_API_KEY env variable
             mkdir test_results
-            poetry run coverage run --include=nucleus/* -m pytest --junitxml=test_results/junit.xml
+            poetry run coverage run --include=nucleus/* -m pytest -s -v --junitxml=test_results/junit.xml
```

Comment:
Made the tests more verbose so they don't timeout as easily when step functions is slow about scheduling.